src/idl_gen_rust.cpp

```diff
@@ -1740,6 +1744,18 @@ class RustGenerator : public BaseGenerator {
     code_ += "";
   }
 
+  void GenNamespaceImports() {
```

Comment:
While we're here -- would you add a parameter to this function that inserts `n` spaces to each line, then use it appropriately in the various places `GenNamespaceImports` is called? You can see that in the code from before this PR, the imports were prefixed by spaces.

Reply:
@rw Yes, I forgot to ask you about that.